### PR TITLE
Describe the payment gateway method for the stores

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request describes the payment gateway method for the stores

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/payment/paypal.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=Payment%20Gateway

### CODE REVIEW

File: src/payment/paypal.md [hunk 1]
```diff
@@ -46,6 +46,15 @@ _Configuration - PayPal all-in-one payment solutions_
 
 ## PayPal payment gateways
 
+A payment gateway is an intermediary between customer and bank, it's an electronic payment service that authorizes credit and debit card transactions. Payment gateways are available in online and offline environments. Payments can be accepted by phone, online, or through a mobile app. The transaction is sent to the service provider's processing system and then sent to the customer's bank for verification and confirmation. If verified, the merchant receives the payment without having direct contact with his client's bank account.
```
Comment (shrielenee) [inline src/payment/paypal.md:49]:
```suggestion
A payment gateway is a merchant service provided by an e-commerce application service provider that authorizes credit card or direct payments processing. They work as intermediaries between customers and banks.

Payment gateways are available in online and offline environments. Payments can be accepted by phone, online, or through a mobile app. The transaction is sent to the service provider's processing system and then sent to the customer's bank for verification and confirmation. If verified, the merchant receives the payment without having direct contact with the customer's bank account.
```

File: src/payment/paypal.md [hunk 2]
```diff
@@ -46,6 +46,15 @@ _Configuration - PayPal all-in-one payment solutions_
...
+
+There are two types of payment gateways - direct and hosted.
```
Comment (shrielenee) [inline src/payment/paypal.md:51]:
```suggestion
There are two types of payment gateways---direct and hosted.
```

File: src/payment/paypal.md [hunk 3]
```diff
@@ -46,6 +46,15 @@ _Configuration - PayPal all-in-one payment solutions_
...
+
+- Direct payment gateway allows users to enter their card details in the store itself.
```
Comment (shrielenee) [inline src/payment/paypal.md:53]:
```suggestion
- Direct payment gateways allow users to enter their card details on the store website.
```

File: src/payment/paypal.md [hunk 4]
```diff
@@ -46,6 +46,15 @@ _Configuration - PayPal all-in-one payment solutions_
...
+- Hosted payment gateway redirects user to hosted payment page.
```
Comment (shrielenee) [inline src/payment/paypal.md:54]:
```suggestion
- Hosted payment gateways redirect users to a hosted payment page, outside of the store website.
```

**Review by shrielenee — CHANGES_REQUESTED** [review]

**Review by shrielenee — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1448`.

**Pull request merged**